docs/src/testing/fork-testing.md

```diff
@@ -30,6 +30,10 @@ enum BlockTag {
 }
 ```
 
+> 📝 **Note**
```

Comment:
I think two lines above we have type including pending. Also I think it will be enough to not list it instead of explicitly excluding it, it is harder to maintain as we then should list everything we don't include.